core/templates/dev/head/components/answer_group_editor_directive.html

```diff
@@ -93,9 +93,8 @@
                   suppress-warnings="suppressWarnings()">
   </outcome-editor>
 
-  <br ng-if="answerGroupIsDefault && isCorrectnessFeedbackEnabled">
-  
-  <div ng-if="answerGroupIsDefault && isCorrectnessFeedbackEnabled">
+  <div ng-if="answerGroupIsDefault && isCorrectnessFeedbackEnabled()">
+    <br>
```

Comment:
Using a margin is still better practice than `<br>`. It's more controllable.

Reply:
Done!